src/utils/variables.scss

```diff
@@ -0,0 +1,16 @@
+$-color-background: rgb(112, 166, 224);
```

Comment:
Consider using more descriptive variable names. `$-color-background` is not very descriptive. Consider using something like `$primary-color` or `$background-color`. 🎨